### DISCUSSION TITLE
Meldung "Wechselrichter gestartet" aktualisiert sich nicht mehr

### DISCUSSION THREAD

**Discussion by NintendoTito** [tbnobody/OpenDTU#1817] [Q&A] [answered]
Guten Morgen zusammen!

Meine OpenDTU funktioniert grundsätzlich, Daten werden aktualisiert, alles bestens. Manchmal schaue ich in die (meist einzige) Benachrichtigung des Tages. Diese zeigt an, wann der WR gestartet ist. Und hier steht schon seit ein paar Tagen "07:32 Uhr Wechselrichter gestartet".

Definitiv ist der WR in den letzten Tagen schon eher gestartet, die Meldung wird nicht mehr aktualisiert. Kommt diese Meldung vom WR, also ist die auf dem WR gespeichert? Oder ist das ne Meldung der DTU? Ist das Problem bekannt?

**Comment by MetaChuh** [comment]
@NintendoTito 

ja, die meldung(en) kommt(kommen) alleine aus dem wr, ohne bezug auf den tag, nur mit uhrzeit.

normalerweise passiert das von dir beschriebene nur, wenn ein wr 24/7 mit dc eingangsstrom versorgt wird (batterie) und nicht jeden tag um 23:59 per api oder mqtt befehl neu gestartet wird.

dann ist aus sicht des inverters der tag nicht vorbei, und er zeigt als endzeit der message 00:00:00 an

**Reply by NintendoTito** [reply]
Mmh, ok. Ne Batterie habe ich definitiv nicht dran. Einfach nur zwei Module. Gefühlt würde ich sagen, dass das Verhalten im Laufe dieser Woche begann. Plötzlich veränderte sich die Uhrzeit nicht mehr. Habe in der Zeit auch kein Update von OpenDTU vorgenommen.

**Reply by MetaChuh** [reply, marked as answer]
danke für die zusatzinfo.

falls du's noch nicht versucht hast:
1) trenne heute nacht den wechselrichter vom ac netz.
(frühestens wenn die inverter led nicht mehr rot oder grün blinkt. ideal: ac getrennt und dc mc4 ausgesteckt)

2) trenne die opendtu von der usb stromversorgung.

3) warte sicherheitshalber 10 minuten (vorsichtsmaßname, normalerweise reicht weniger)

4) verbinde den inverter wieder mit dem ac netz (sowie panels, falls auch ausgesteckt)

5) verbinde die opendtu wieder mit der usb stromversorgung.

6) berichte uns von dem ergebnis morgen früh.

anmerkung: die reihenfolge vom wieder verbinden in  den punkten 4) und 5) spielt normalerweise keine rolle

**Reply by NintendoTito** [reply]
Gestern gegen 21 Uhr die DTU vom Strom genommen und gleichzeitig die DEN Wechselrichter übers ausschalten des Shelly vom Netz genommen. Heute morgen dann zwei Meldungen in OpenDTU. Einmal die aktuelle Startzeit gegen 6:57 Uhr und eine Meldung "Time calibration". Scheint also alles wieder zu laufen.

**Reply by MetaChuh** [reply]
@NintendoTito 

super, freut uns das zu lesen 👍
vielen dank für dein gutes feedback 🙏👍

greetings,
metachuh
